adapters/oidc/adapter-core/src/main/java/org/keycloak/adapters/AuthenticatedActionsHandler.java

```diff
@@ -118,15 +118,17 @@ protected boolean corsRequest()  {
         if (securityContext != null && origin != null && !origin.equals(requestOrigin)) {
             AccessToken token = securityContext.getToken();
             Set<String> allowedOrigins = token.getAllowedOrigins();
-            if (log.isDebugEnabled()) {
+
+            if (log.isDebugEnabled() && allowedOrigins != null) {
```

Comment:
Why not replace the whole thing with
```java
log.debugf("Allowed origins in token: %s", allowedOrigins);
```
There is no need for `if` then, it's then less error-prone.